src/Umbraco.Web.UI.Client/src/views/languages/edit.html

```diff
@@ -21,12 +21,15 @@
                 <umb-box ng-if="!vm.loading" class="block-form">
                     <umb-box-content>
 
-                        <umb-control-group label="@general_language">
-                            <select 
-                                class="input-block-level"
-                                ng-model="vm.newLanguage.culture"
-                                ng-options="key as value for (key, value) in vm.availableCultures">
-                            </select>
+                        <umb-control-group label="@general_language" ng-if="vm.availableCultures != null">
```

Comment:
The `!=null` is not needed as null & undefined & empty array all return falsely @Shazwazza 
I will fix it up as super trivial